openedx/adg/lms/urls.py

```diff
@@ -18,6 +18,11 @@
         r'^api/applications/',
         include('openedx.adg.lms.applications.api_urls', namespace='applications_api')
     ),
+    # ADG webinars app
+    url(
+        r'^webinar/',
```

Comment:
Do we want to name it `webinars` or `webinar`? As the app is named `webinars`

Reply:
What do you suggest, currently it is the first one

```
/webinar/{id}
/webinar/{id}/register
/webinar/{id}/cancel
```

or 

```
/webinars/{id}
/webinars/{id}/register
/webinars/{id}/cancel
```

Reply:
Let's go with the second option as [Django official docs](https://docs.djangoproject.com/en/3.2/intro/tutorial03/#writing-more-views) also uses plural form.

Secondly, it is a better option if we are going to have a url that lists all the webinars.
```
/webinars (lists all the webinars)
/webinars/{id}
/webinars/{id}/register
/webinars/{id}/cancel
```

Reply:
done